2_Operations/user-stories.md

```diff
@@ -20,45 +20,45 @@ Whenever possible, think about the end user (product persona) and how the result
+Write technical details on how to accomplish this proposal. Add references to any previous discussions of the proposal. Provide links to related information, such as existing User Stories. Assign anyone that you need feedback from, for this User Story.
 
 Finally, take a bit of time to consider at least one pitfall (downside) to implementing the proposed changes. So what could go wrong or how could it affect other systems?
 
 ### Best practices (Definition of Done)
 
-The Best Practices are a list of items that should relate to the story and guide the writing of the Expectations for completion of the story.
+The Best Practices are a list of items that should relate to the Story and guide the writing of the Expectations for completion of the Story.
 
 All major features should have a blog post written. If you are not sure if it should be written or not, ask the Scrum master or product owner. The marketing team writes and reviews the details of the actual blog post.
 
 ### Expectations (Acceptance Criteria)
 
-The expected outcomes of the user story which is a list of items stating how the world will look once the user story has been completed. This is not the individual task or activities but rather what will the documentation look like, what new tests will we see, what bug no longer occurs because it has been fixed.
+The expected outcomes of the User Story which is a list of items stating how the world will look once the User Story has been completed. This is not the individual task or activities but rather what will the documentation look like, what new tests will we see, what bug no longer occurs because it has been fixed.
 
 ## Story Points
 
-A [Story Point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story. One full-time member on sprint should be able to do 10 Story Points.
+A [Story Point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-Story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story. One full-time member on sprint should be able to do 10 Story Points.
 
 ## Pipelines
 
-From initial creation to being worked upon and finally closed, a user story will be moved to different pipelines as it progresses through a sprint. This is a summary of the workflow of a user story:
+From initial creation to being worked upon and finally closed, a User Story will be moved to different pipelines as it progresses through a sprint. This is a summary of the workflow of a User Story:
 
 1. **New Issues** - New work-in-progress (WIP) user stories, also includes bugs or issues.
-1. **Estimation** - Defined user stories, ready for verifying and story point estimation.
-1. **Backlog** - Waiting for inclusion in a sprint. They are adequately defined with assigned story points.
+1. **Estimation** - Defined user stories, ready for verifying and Story point estimation.
```

Comment:
Consistency. Shouldn't `user stories` be -> `User Stories`?

Reply:
Yep, search and replace doesn't know plural :)